Suggestion for StringComparison.InvariantCultureIgnoreCase is wrong ?

Suggestion is like:.

_System.StringComparer      get_InvariantCultureIgnoreCase_
Use ```CompareInfo.GetStringComparer```

However this has an error?
```D:\Projects\dotnet\Program.cs(13,46): error CS0117: 'CompareInfo' does not contain a definition for 'GetStringComparer'```

--- a/docs/RecommendedChanges/System/Use CompareInfo.GetStringComparer.md
+++ b/docs/RecommendedChanges/System/Use CompareInfo.GetStringComparer.md
@@ -1,5 +1,5 @@
 ### Recommended Action
-Use CompareInfo.GetStringComparer.
+Use `System.Globalization.CultureInfo.InvariantCulture.CompareInfo.GetStringComparer(CompareOptions.IgnoreCase)` from the `System.Globalization.Extensions` package.
 
 ### Affected APIs
 * `M:System.StringComparer.Create(System.Globalization.CultureInfo,System.Boolean)`

